ROOTFS/etc/waggle/sanity/fatal/core_sdcard.test

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+pf="Recovery SDCard test"
+echo "$pf: Begin"
+
+trap "if mountpoint -q "/tmp/core_sdcard_test"; then umount /tmp/core_sdcard_test; fi" EXIT
```

Comment:
do you also want to `rm -r /tmp/core_sdcard_test/`

also lets make this trap a function call... as your overloaded use of `"` is confusing... does it even work?